utils/baker.py

```diff
@@ -15,6 +15,13 @@
 from .render import set_guide_height, get_rendered_objects
 from .scene import scene_setup
 
+def BlenderVersionEevee ()-> str:
+    #print (str(bpy.app.version))
+    if bpy.app.version > (4, 1, 0) :
```

Comment:
This would work better as if `bpy.app.version >= (4, 2, 0):`, as how the code is currently written would cause an error with Blender 4.1.1+